root/static/scripts/release-editor/seeding.js

```diff
@@ -79,8 +81,8 @@
         }
 
         if (data.artistCredit) {
-            release.artistCredit.setNames(data.artistCredit);
-            release.artistCredit.saved = fields.ArtistCredit(data.artistCredit);
+            release.artistCredit(artistCreditFromArray(data.artistCredit));
+            release.artistCredit.saved = release.artistCredit.peek();
```

Comment:
Since the AC objects are immutable we can just pass the same reference around everywhere.